source/kraken/apply_disruption.cpp

```diff
@@ -284,6 +359,9 @@ struct add_impacts_visitor : public apply_impacts_visitor {
                 new_vj->physical_mode = pt_data.physical_modes[0];
                 new_vj->name = new_vj_uri;
             }
+            /*
+             * Properties manually added to guarantee the good behavior for raptor.
```

Comment:
Not only. For example, raptor don't care of the company, but that's quite coherent to say that the company doesn't change.